{
    "url": "https://api.github.com/repos/pallets/flask/issues/192",
    "repository_url": "https://api.github.com/repos/pallets/flask",
    "labels_url": "https://api.github.com/repos/pallets/flask/issues/192/labels{/name}",
    "comments_url": "https://api.github.com/repos/pallets/flask/issues/192/comments",
    "events_url": "https://api.github.com/repos/pallets/flask/issues/192/events",
    "html_url": "https://github.com/pallets/flask/issues/192",
    "id": 647415,
    "node_id": "MDU6SXNzdWU2NDc0MTU=",
    "number": 192,
    "title": "log exceptions using app.logger",
    "user": {
        "login": "orutherfurd",
        "id": 411934,
        "node_id": "MDQ6VXNlcjQxMTkzNA==",
        "avatar_url": "https://avatars1.githubusercontent.com/u/411934?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/orutherfurd",
        "html_url": "https://github.com/orutherfurd",
        "followers_url": "https://api.github.com/users/orutherfurd/followers",
        "following_url": "https://api.github.com/users/orutherfurd/following{/other_user}",
        "gists_url": "https://api.github.com/users/orutherfurd/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/orutherfurd/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/orutherfurd/subscriptions",
        "organizations_url": "https://api.github.com/users/orutherfurd/orgs",
        "repos_url": "https://api.github.com/users/orutherfurd/repos",
        "events_url": "https://api.github.com/users/orutherfurd/events{/privacy}",
        "received_events_url": "https://api.github.com/users/orutherfurd/received_events",
        "type": "User",
        "site_admin": false
    },
    "labels": [],
    "state": "closed",
    "locked": false,
    "assignee": null,
    "assignees": [],
    "milestone": null,
    "comments": 2,
    "created_at": "2011-03-04T18:03:49Z",
    "updated_at": "2015-11-03T14:55:10Z",
    "closed_at": "2015-11-03T14:55:10Z",
    "author_association": "NONE",
    "body": "I'm working on a project where we're running flask w/mod_wsgi in development and it'd be really nice to be able to route all errors the app's log (vs having some in the app's log and some in the apache log).  Though the built-in dev server is more convenient, we're running mod_wsgi in development for consistency with production.\n",
    "comments_inline": [
        {
            "url": "https://api.github.com/repos/pallets/flask/issues/comments/1751193",
            "html_url": "https://github.com/pallets/flask/issues/192#issuecomment-1751193",
            "issue_url": "https://api.github.com/repos/pallets/flask/issues/192",
            "id": 1751193,
            "node_id": "MDEyOklzc3VlQ29tbWVudDE3NTExOTM=",
            "user": {
                "login": "Poincare",
                "id": 217352,
                "node_id": "MDQ6VXNlcjIxNzM1Mg==",
                "avatar_url": "https://avatars2.githubusercontent.com/u/217352?v=4",
                "gravatar_id": "",
                "url": "https://api.github.com/users/Poincare",
                "html_url": "https://github.com/Poincare",
                "followers_url": "https://api.github.com/users/Poincare/followers",
                "following_url": "https://api.github.com/users/Poincare/following{/other_user}",
                "gists_url": "https://api.github.com/users/Poincare/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/Poincare/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/Poincare/subscriptions",
                "organizations_url": "https://api.github.com/users/Poincare/orgs",
                "repos_url": "https://api.github.com/users/Poincare/repos",
                "events_url": "https://api.github.com/users/Poincare/events{/privacy}",
                "received_events_url": "https://api.github.com/users/Poincare/received_events",
                "type": "User",
                "site_admin": false
            },
            "created_at": "2011-08-08T02:39:47Z",
            "updated_at": "2011-08-08T02:39:47Z",
            "author_association": "NONE",
            "body": "If anyone agrees to doing this, I can start working on this.\n"
        },
        {
            "url": "https://api.github.com/repos/pallets/flask/issues/comments/4465393",
            "html_url": "https://github.com/pallets/flask/issues/192#issuecomment-4465393",
            "issue_url": "https://api.github.com/repos/pallets/flask/issues/192",
            "id": 4465393,
            "node_id": "MDEyOklzc3VlQ29tbWVudDQ0NjUzOTM=",
            "user": {
                "login": "aisipos",
                "id": 68444,
                "node_id": "MDQ6VXNlcjY4NDQ0",
                "avatar_url": "https://avatars0.githubusercontent.com/u/68444?v=4",
                "gravatar_id": "",
                "url": "https://api.github.com/users/aisipos",
                "html_url": "https://github.com/aisipos",
                "followers_url": "https://api.github.com/users/aisipos/followers",
                "following_url": "https://api.github.com/users/aisipos/following{/other_user}",
                "gists_url": "https://api.github.com/users/aisipos/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/aisipos/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/aisipos/subscriptions",
                "organizations_url": "https://api.github.com/users/aisipos/orgs",
                "repos_url": "https://api.github.com/users/aisipos/repos",
                "events_url": "https://api.github.com/users/aisipos/events{/privacy}",
                "received_events_url": "https://api.github.com/users/aisipos/received_events",
                "type": "User",
                "site_admin": false
            },
            "created_at": "2012-03-13T00:53:26Z",
            "updated_at": "2012-03-13T00:53:26Z",
            "author_association": "CONTRIBUTOR",
            "body": "Since Flask 0.8,  if exceptions are not set to propagate, unhandled exceptions are sent to the application error log. Arguably, this should not be part of Flask itself, since applications can't control the formatting of the exception. \n\nThe docs on error handling could use some updating on this:\nhttp://flask.pocoo.org/docs/errorhandling/\n"
        }
    ]
}